src/docs/product/sentry-basics/search/searchable-properties/issues.mdx

```diff
@@ -26,13 +26,13 @@ Indicates if the app is in the foreground or background. Values are `1/0` or `tr
 
 ### `assigned`
 
-Returns issues assigned to the defined user. Values can be a user ID (your email address), `me` for yourself, `me_or_none` for yourself or no assignee, or # followed by the team name, that is `#team-name`.
+Returns issues assigned to the defined user(s) or team(s). Values can be a user ID (your email address), `me` for yourself, `me_or_none` for yourself or no assignee, `my_teams` or `#team-name` for teams you belong to.
```

Comment:
Hmm, `me_or_none` might be confusing for the user to use. I'm not sure thats a legal value a user can enter. 

Would something like:
```
Values can be a user ID (your email address), `me` for yourself, `my_teams` for teams you belong to, `#team-name` for a specific team assigned to issues, or `none` for issues that aren't assigned.
```